packages/flutter/lib/src/services/platform_messages.dart

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 The Chromium Authors. All rights reserved.
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
nit: (don't fix this, but for future knowledge): don't change the copyright dates.